### PR TITLE
Top 10 book review api doc

### PR DESCRIPTION
**Pull request by ghost**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: #PBI-35
- Task ID: #35.1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: docs/api-references/books.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Get list books with paginate
 ```json
 {
     "meta": {
-        "status" : "success",
+        "message" : "success",
```
Comment (at-annguyen) [inline docs/api-references/books.md]: message là để show msg trả về chứ k phải success or fail. status -> successfully

File: docs/api-references/books.md [hunk 2]
```diff
@@ -49,3 +49,85 @@ Get list books with paginate
...
+        },
+        {
+            "name": "Luật trí não",
+            "image": "booklibrary.test/images/books/default/no-image.png",
+            "avg_rating": 0,
+            "posts_count": 2
+        }
+    ]
+}
+```
```
Comment (at-annguyen) [inline docs/api-references/books.md]: có trường hợp trả về failure không?

Comment (at-anhngo) [reply docs/api-references/books.md]: get thì chắc chắn thành công mà nên không cần failure đâu

**Comment by at-annguyen** [comment]
LGTM

**Comment by at-annguyen** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**